log-collector/src/main/scala/io/phdata/pulse/logcollector/LogCollector.scala

```diff
@@ -43,8 +45,9 @@ object LogCollector extends LazyLogging {
 
     val cliParser = new LogCollectorCliParser(args)
 
-    val solrServer  = new CloudSolrServer(cliParser.zkHosts())
-    val solrService = new SolrService(cliParser.zkHosts(), solrServer)
+    val solrServer      = new CloudSolrServer(cliParser.zkHosts())
```

Comment:
These solr definitions should go inside of the 'serve' function so the connections aren't instantiated unless they are going to be used.

Reply:
@afoerster I moved the solrCloudStream instantiation into 'consume' as it is only used there. but shouldn't solrServer and solrService stay here since both 'serve' and 'consume' need them?